omnibus/package-scripts/ddot-deb/postinst

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+##########################################################################
+#             DO NOT EDIT THIS SCRIPT DIRECTLY.                          #
+#                                                                        #
+# The installation logic is handled by the installer at in the following #
+# file: pkg/fleet/installer/packages/datadog_agent_ddot_linux.go         #
+#                                                                        #
+##########################################################################
+
+# If we are inside the Docker container, do nothing
+if [ -n "$DOCKER_DD_AGENT" ]; then
+    echo "Installation from docker-dd-agent, nothing to do in postinst"
+    exit 0
+fi
+
+# Run the postinst. See pkg/fleet/installer/packages/datadog_agent_ddot_linux.go
+datadog-installer postinst datadog-agent-ddot deb || true
+
+exit 0
```

Comment:
minor: redundant (many occurrences). Or is it by convention?

Reply:
This is here because we mirrored the files in `agent-deb` and `agent-rpm`